transformer_engine/pytorch/module/linear.py

```diff
@@ -335,7 +343,7 @@ def forward(
                 weight,
                 weight_fp8,
                 weight.main_grad if cpu_offloading and fuse_wgrad_accumulation else None,
-                fp8_meta["scaling_fwd"].scale_inv.clone() if fp8 else None,
+                inputmat_fp8_scale_inv,
```

Comment:
This is only set for float8tensor case

Reply:
We handle three cases:
- Non-FP8: scale-inv is `None`
https://github.com/NVIDIA/TransformerEngine/blob/23258bb400a4bdb0dba6e741f66e104cdf8b6032/transformer_engine/pytorch/module/linear.py#L120
- FP8, `Float8Tensor` input: scale-inv is taken from `Float8Tensor`
https://github.com/NVIDIA/TransformerEngine/blob/23258bb400a4bdb0dba6e741f66e104cdf8b6032/transformer_engine/pytorch/module/linear.py#L125
- FP8, non-`Float8Tensor` input: scale-inv is computed with fast kernel
https://github.com/NVIDIA/TransformerEngine/blob/23258bb400a4bdb0dba6e741f66e104cdf8b6032/transformer_engine/pytorch/module/linear.py#L158